app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -57,12 +59,14 @@ open class HomeActivity : AppCompatActivity() {
         setSupportActionBar(navigationToolbar)
         NavigationUI.setupWithNavController(navigationToolbar, hostNavController, appBarConfiguration)
 
-        handleOpenedFromExternalSourceIfNecessary(intent)
-    }
+        val safeIntent = intent?.let { SafeIntent(it) }
 
-    override fun onResume() {
-        super.onResume()
-        components.analytics.metrics.track(Event.OpenedApp)
+        if (safeIntent?.isLauncherIntent == true) {
+            val source = if (isCustomTab) Event.OpenedApp.Source.CUSTOM_TAB else Event.OpenedApp.Source.APP_ICON
```

Comment:
Isn't there more than two possibilities? We could've been opened by a VIEW intent, for example.